### PR TITLE
Add Review + Edge Cases + Profanity Censor

### PR DESCRIPTION
**Pull request by AdnanAhsanNYCity**

### CODE REVIEW

File: backend/frontend/src/components/GoogleMapContainer.jsx [hunk 1]
```diff
@@ -197,7 +247,54 @@ export const GoogleMapContainer = (props) => {
...
+                                min={1}
+                                max={5}
+                            />
+                            </label>
+                            <label>Review:
+                                <input 
+                                    type="text" 
+                                    name="review" 
+                                    value={inputs.review || ""} 
+                                    required = "required"
```
Comment (ds6332) [inline backend/frontend/src/components/GoogleMapContainer.jsx:275]: lgtm. 
Is the review "required" though?

**Review by ds6332 — APPROVED** [review]
lgtm

File: backend/frontend/src/components/GoogleMapContainer.jsx (hunk 1 above)
Comment (guptaviha) [reply backend/frontend/src/components/GoogleMapContainer.jsx:275]: Agreed. Lets let it pass for now and revisit in a later sprint

**Review by guptaviha — APPROVED** [review]
Look good to merge for now. A couple of pointers for improvement:

Discuss - On the use of profanity, should the review be blocked?
Discuss - Should the review be a required field?
Need to clear state after successful review
On submitting of review, the page should refresh and show the latest review
A successful review msg should be a toast msg rather than an alert
Null checks on rating and review fields should show in-line errors like login form

For now lets merge it to dev

**Review by as14680 — APPROVED** [review]
lgtm

**Pull request merged**